### PR TITLE
Add 1.11 Exceptions List

### PR DESCRIPTION
**Pull request by justaugustus**
/assign @calebamiles @jberkus 

Signed-off-by: Stephen Augustus <foo@agst.us>

### CODE REVIEW

File: releases/release-1.11/exceptions.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Exceptions to code freeze requested in 1.11
+# Google Group: https://groups.google.com/forum/#!topic/kubernetes-milestone-burndown
+# Release Team Lead: Josh Berkus (@jberkus)
+
+- name: "Update Istio addon manifest"
+  issue: "https://github.com/kubernetes/kubernetes/issues/64563"
```
Comment (idvoretskyi) [inline releases/release-1.11/exceptions.yaml]: Needs to be tracked under the k/features repo.

Comment (calebamiles) [reply releases/release-1.11/exceptions.yaml]: please explain, @idvoretskyi. This is intended as a point of record for exceptions to code freeze which have been requested with their resolution

Comment (calebamiles) [reply releases/release-1.11/exceptions.yaml]: That seems independent of whether they had a feature issue open or not although they _should_ have

**Pull request merged**